test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -214,6 +214,12 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
             }
             $service = New-Service @parameters
             $service | Should -Not -BeNullOrEmpty
+
+            if ($displayname) {
+                $service.displayname | Should -Be $displayname
+            }
```

Comment:
> Rather than make this check conditional such that it only runs some of the time, how about just making it check for the appropriate value and having it run all the time?
> 
> ```powershell
> $service.displayname | Should -Be $(if ($displayname) {$displayname} else {$name})
> ```

This will make sure if `DisplayName` null, then `ServiceName` becomes `DisplayName`